experimental/GModule/src/GModule.jl

```diff
@@ -968,6 +1037,65 @@ function natural_gmodule(G::PermGroup, R::Ring)
 #      without storing a matrix?
 end
 
+function show_permutation_gmodule(io::IO, C::GModule)
+  io = pretty(io)
+  io = IOContext(io, :compact => true)
+  rc = get_attribute(C, :right_cosets)
+  print(io, "permutation module for $(rc.group) over $(base_ring(C)) for right cosets of $(rc.subgroup)")
```

Comment:
```suggestion
  print(io, "Permutation module for ", Lowercase(), rc.group, " over ", Lowercase(), base_ring(C), " for right cosets of ", Lowercase(), rc.subgroup)
```
Add appropriate casing